bigquery_etl/shredder/config.py

```diff
@@ -90,12 +89,6 @@ def fields(self) -> Tuple[str, ...]:
     table="telemetry_stable.deletion_request_v4",
     field="payload.scalars.parent.deletion_request_impression_id",
 )
-CFR_SRC = DeleteSource(
-    # inject sql via f"`{sql_table_id(source)}`" to select client_id and impression_id
-    table="telemetry_stable.deletion_request_v4`,"
-    f" UNNEST([{CLIENT_ID}, {IMPRESSION_SRC.field}]) AS `_",
-    field="_",
-)
```

Comment:
this sql injection would break with the `LEFT JOIN` format, and it should to be migrated to the newer format anyway.